src/libraries/System.Private.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/GeneratedInvokeJsTest.part.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices.JavaScript.Private;
+
+namespace System.Runtime.InteropServices.JavaScript.Private
+{
+    // this is handwritten what should be generated
+    public static unsafe class JSMethods
+    {
+        static JavaScriptMarshalerSignatures1* signatureStringVoid = JavaScriptMarshalerSignatures1<string, object>.SignatureVoid;
+        public static IntPtr LogJsHandle = JavaScriptMarshalImpl.BindJSFunction("console.log", signatureStringVoid);
```

Comment:
we should think about what error handling looks like for the scenario where this fails. we might want to initialize all these fields in one generated method, vs using field initializers which will sort of throw an exception 'wherever'

Reply:
For `JSImport` we should be lazy, because that's good for linking it out.
For `JSExport` we could not be lazy, because we don't know when it would be called from JS side. That also means `JSExport` prevents linking out.